applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_adjoint_solver.py

```diff
@@ -124,10 +125,9 @@ def get_response_function(self):
         return self._response_function
 
     def _create_response_function(self):
-        computing_model_part = self.GetComputingModelPart()
         if self.response_function_settings["response_type"].GetString() == "adjoint_lift_jump_coordinates":
             response_function = KCPFApp.AdjointLiftJumpCoordinatesResponseFunction(
-                computing_model_part,
+                self.main_model_part,
```

Comment:
Why is the `main_model_part` required here? Shouldn't be everything computed with the  `computing_model_part`?

Reply:
You are right! When the test were failing I compared to the master and I saw we were using the `main_model_part`, so the first thing I tried was switching back to the` main_model_part`.

I have changed it back to using the `computing_model_part`, which required to also change some test settings, but it is cleaner now!

Reply:
use the function `GetComputingModelPart()`

Reply:
> use the function `GetComputingModelPart()`

It is already used. `computing_model_part` is retrieved through `GetComputingModelPart()`.

Reply:
Ok .. !